module/VuFind/src/VuFind/Search/Combined/Options.php

```diff
@@ -81,6 +81,7 @@ public function getRecommendationSettings($handler = null)
         $recommend = [];
         $config = $this->configLoader->get('combined');
         foreach (['top', 'bottom'] as $location) {
+            // foreach (['top', 'bottom', 'side'] as $location) {
```

Comment:
Is this needed? Only change in this file is this commented-out line.

Reply:
This would be needed if I leave in approach 3 as well.  TBD.